src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/LicenceSummary.kt

```diff
@@ -186,4 +186,11 @@ data class LicenceSummary(
 
   @Schema(description = "The full name of the person who last updated this licence", example = "Jane Jones")
   val updatedByFullName: String? = null,
+
+  @Schema(
+    description = "The person’s actual home detention curfew date",
+    example = "30/01/2025",
+  )
+  @JsonFormat(pattern = "dd/MM/yyyy")
+  val homeDetentionCurfewActualDate: LocalDate? = null,
```

Comment:
This is where not having different subtypes for different licence summaries lets us down. 

Worth mentioning, this will always be null for non HDC licences?